src/components/ViewRequest/ViewRequest.jsx

```diff
@@ -0,0 +1,154 @@
+import { useEffect, useState } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useBackend } from '../../contexts/BackendContext';
+import {
+  Box,
+  Button,
+  Card,
+  ChakraProvider,
+  Flex,
+  Table,
+  Heading,
+  Thead,
+  Tbody,
+  Th,
+  Tr,
+  Td,
+} from '@chakra-ui/react';
+
+const ViewRequest = () => {
+  const { backend } = useBackend();
+  const { id } = useParams();
+  const navigate = useNavigate();
+  const supplyItems = [
+    'Get Pet Food Decal',
+    'Decal',
+    '"Homeless?" Card',
+    'Business Card',
+    'Donation Site Decal',
+    'Donation Site Bin Decal',
+    'Donation Envelopes',
+    '"Homeless?" Card 2',
+  ];
+  const items = [
+    'Get Pet Food Decal',
+    'Decal',
+    'Homeless Card',
+    'Business Card',
+    'Donation Site Decal',
+    'Donation Site Bin Decals',
+    'Donation Envelopes',
+    'Homeless Card 2',
+  ];
+  const [itemAmounts, setItemAmounts] = useState({});
+  const [businessName, setBusinessName] = useState('');
+  const [dateRequested, setDateRequested] = useState('');
+
+  function formatDate(dateTimeString) {
+    const date = new Date(dateTimeString);
+    const options = {
+      year: 'numeric',
+      month: '2-digit',
+      day: '2-digit',
+    };
+    return date.toLocaleDateString('en-US', options);
+  }
+
+  const fetchRequest = async () => {
+    try {
+      const response = await backend.get(`/notification/request/${id}`);
+      const message = response.data[0].message;
+      setItemAmounts(JSON.parse(message));
+      setDateRequested(formatDate(response.data[0].timestamp));
+
+      const businessNameResponse = await backend.get(`/business/${response.data[0].sender_id}`);
+      setBusinessName(businessNameResponse.data[0].name);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+  useEffect(() => {
+    fetchRequest();
+  }, [id]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'fetchRequest'. Either include it or remove the dependency array.